patterns/vsi/module/variables.tf

```diff
@@ -530,3 +530,15 @@ variable "IC_SCHEMATICS_WORKSPACE_ID" {
 }
 
 ##############################################################################
+
+##############################################################################
+# CBR variables
+##############################################################################
+
+variable "existing_vpc_cbr_zone_id" {
+  type        = string
+  description = "Existing VPC CBR zone id"
```

Comment:
Perhaps explain a little more about what happens when user passes a value for this